src/vehicle.h

```diff
@@ -716,16 +716,20 @@ class vehicle : public JsonSerializer, public JsonDeserializer
     // 1.0 means mass won't slow vehicle at all, 0 - it won't move
     float k_mass () const;
 
+    // Coefficient of traction
+    // Result depends on wheel traction, wheel area and vehicle's total mass
```

Comment:
Best as doxygen comments. Should also specify `@return` outlining possible range of return values